src/core/layout/footer/styles.jsx

```diff
@@ -1,9 +1,9 @@
 import { makeStyles } from '@material-ui/core'
 
 const position = theme => ({
-  position: 'absolute',
-  width: '100vw',
   bottom: theme.spacing(0),
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125439160-83aecc60-7b6d-4bd5-9dd7-3a1479d0c03a.png)
At first I thought this `title` refers to the `title` component, not the message present in the footer. :smile: 

Let's rename it to something else to avoid confusion in the future.

Reply:
Also, let's rename the `footer` field to `root`, so that we are in line with the `Material-UI` standard.

Reply:
Done